bokeh/enums.py

```diff
@@ -42,3 +42,4 @@ def enumeration(*values):
 NamedColor = enumeration(*colors.__colors__)
 NamedIcon = enumeration(*icons.__icons__)
 MapType = enumeration("satellite", "roadmap", "terrain", "hybrid")
+DateFormat = enumeration("ATOM", "W3C", "RFC-3339", "ISO-8601", "COOKIE", "RFC-822", "RFC-850", "RFC-1036", "RFC-1123", "RFC-2822", "RSS", "TICKS", "TIMESTAMP")
```

Comment:
We generally adhere to PEP8... but sometimes is reasonable to break the recommendations...
In this specific change, I see value in both views... so I have not strong feelings for any of the positions.